js/client/modules/@arangodb/testutils/test-utils.js

```diff
@@ -173,7 +175,10 @@ function performTests (options, testList, testname, runFn, serverOptions, startS
   let count = 0;
   let forceTerminate = false;
   let graphCount = 0;
-
+  let usersCount = 0;
+  if (testname !== 'agency') {
```

Comment:
Why are we excluding `agency` here, and `agency` and `authentication` further down?

Reply:
agency has no authentification at all, so this will throw. `authentication` was excluded feature wise, but could be added here as well.